wiki/Article_Style_Guide/en.md

```diff
@@ -101,28 +119,35 @@ or
 You can choose either the underline style or hashtag style of level 1 and level 2 headings.
 Note that heading levels 3 and onwards use the hashtag style.
 
+Optionally, but preferably, add an extra line before and after the headings.
+Doing so will help make the headings stand out a bit more when editing.
```

Comment:
The `news` dump is exempt from the ASG, it is an entirely different entity, if anything; unless something happened that I was unaware of or there is something that I overlooked...?